### ISSUE TITLE
Mid-Atlantic Doppio Bottom Layer Unclear Depth Slider

### ISSUE THREAD

**Issue by jamesdoyle21** [asascience/ott-coops#335]
It is unclear what this depth slider represents

![image](https://user-images.githubusercontent.com/82955395/152852187-5c3a6431-19f2-43dc-9275-bd5bbe80f197.png)

**Comment by mpiannucci** [comment]
Can we get clarification of what you mean by this?

**Comment by jamesdoyle21** [comment]
Yeah so the 3D depth slider for this layer runs between 0 and -1 meters. What do these numbers represent? Is -1 the bottom and 0 is 1m above the bottom? Vice versa? Is this at the surface? The may be a question for @cpaternostro

**Comment by jamesdoyle21** [comment]
Ok, so is the depth slider showing up a bug? Is this not a 3D layer at all?

**Comment by cpaternostro** [comment]
Depth sliders between 0-1 are likely showing the zeta levels.  These are
model reference layers, as a proportion of the total depth at each point.
e.g.,  0.5 zeta is half way in the water column for all areas regardless of
the different depths at the different points.

--

Chris Paternostro

https://tidesandcurrents.noaa.gov/

**Comment by jamesdoyle21** [comment]
If this layer is supposed to be sigma, the depth slider needs to be changed. It reads in meters now.

**Comment by kvmartin001** [comment]
@jamesdoyle21 Based on the Doppio ROMS layer abstract and [this paper about it](https://gmd.copernicus.org/articles/13/3709/2020/), I think we can clarify this by renaming:

- Mid Atlantic (Doppio ROMS) -> Mid Atlantic (Doppio ROMS, Surface Only)
- Mid Atlantic (Doppio Bottom) -> Mid Atlantic (Doppio ROMS 3D)

Both of these layers come from a model that is using sigma for depth.
In the MARACOOS catalog, these two layers have some confusing quirks:

1. The "Doppio ROMS" layer is from PyXms, contains the full abstract for the Doppio model, but only includes one depth (surface)
2. The "Doppio Bottom" layer is from EDS, has no abstract, and includes data at **all** depths. It just happens to default to 0 sigma (seafloor). If you slide the depth slider back up to -1, you will get the surface layer data.
3. The elevation extents of "Doppio Bottom" have a blank unit identifier, so OM3 wrongly detects it as meters instead of sigma.
4. The default colorscale for "Doppio Bottom" is also at 0 to 100C, which is deceiving because it makes the surface data look super cold (as if it were bottom). Default should be 0 to 35C like the other temperature layers.
